docs/DEVELOPMENT-OSX-NATIVE.md

```diff
@@ -263,28 +251,44 @@ make install
 ## Setting up your Discourse
 
 ###  Check out the repository
-
-    git@github.com:discourse/discourse.git ~/discourse
-    cd ~/discourse # Navigate into the repository, and stay there for the rest of this how-to
-
+```sh
+git clone git@github.com:discourse/discourse.git
+cd discourse # Navigate into the repository, and stay there for the rest of this how-to
+```
 ### What about the config files?
 
 If you've stuck to all the defaults above, the default `discourse.conf` and `redis.conf` should work out of the box.
 
 ### Install the needed gems
+```sh
+bundle install
+```
 
-    bundle install # Yes, this DOES take a while. No, it's not really cloning all of rubygems :-)
+You may have issues installing therubyracer when running `bundle install`
+because of a dependency on libv8. This is how to fix it:
 
-### Prepare your database
+```sh
+brew tap homebrew/versions
+brew uninstall v8
+brew install v8-315
+gem uninstall -a libv8
+gem uninstall -a therubyracer
+gem install libv8 -v '3.16.14.13' -- --with-system-v8
+gem install therubyracer -v '0.12.2' -- --with-v8-dir=$(brew --prefix v8-315)
```

Comment:
this can be removed and the libv8 stuff ... we don't use therubyracer anymore and libv8 works fine on mac these days anyway.